scripts/run_commands

```diff
@@ -41,8 +41,7 @@ conda clean -tipy
 # Install conda build and deployment tools.
 conda install --yes --quiet \
     git patch \
-    python=3.8 setuptools conda-build anaconda-client \
```

Comment:
Please keep an explicit pin like `python=3.9` here.

Reply:
Added

Though it's worth noting this has a habit of going stale. Would it make sense to merely pin the version of `python` we use to match the version in the installer? We could do the same thing we do with `tini` and `su-exec` for example.